added vscode debugging setup. (#5)
Depado/pb-templ-htmx-tailwind: README.md

Once these two requirements are met:

```sh
$ make dep
$ make
```

> ### VSCODE debug
> 
> The .vscode folder has a configured launch to Run or Debug the golang.
> 

Reviewer: ```suggestion
### Vscode debug

The `.vscode` folder is configured to add a Run and a Debug launch commands.

```
